assignments/arrays.js

```diff
@@ -65,19 +65,42 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
 console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
+let carThirtyThree = inventory[32];
+
+console.log(carThirtyThree);
+
+carThirtyThree.description = function(){
+    return 'This vehicle is a ' + this.car_year + this.car_make + this.car_model + 'and is car number' + this.id + ' on our lot today';
+};
+
+console.log(carThirtyThree.description());
+
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = inventory[49];
```

Comment:
Same issue here.  How would you solve the problem if you did not know the index of the last item?  It should not be expected that the owner knows the exact number of cars he has in his inventory.